src/main/java/core/basesyntax/CollorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public interface CollorSupplier {

```

Comment:
I think that an interface without abstract methods makes no sense, let's make CollorSupplier as  class